src/golang/lib/execution_environment/dependency.go

```diff
@@ -3,7 +3,7 @@ package execution_environment
 import (
 	"archive/zip"
 	"bytes"
-	"io/ioutil"
```

Comment:
Linter complaint:
```
lib/execution_environment/dependency.go:6:2: SA1019: "io/ioutil" has been deprecated since Go 1.16: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details. (staticcheck)
        "io/ioutil"
        ^
```